export-or-backup-using-dumpling.md

```diff
@@ -0,0 +1,122 @@
+summary: Use the Dumpling tool to export or backup data in TiDB.
+category: how-to
+---
+
+# Export or Backup Data Using Dumpling
+
+This document introduces how to use the [Dumpling](https://github.com/pingcap/dumpling) tool to export or backup data in TiDB. Dumpling exports data stored in TiDB as SQL or CSV data files and can be used to complete logical full backup or export.
+
+For backups of SST files (KV pairs) or backups of incremental data that are not sensitive to latency, refer to [BR](/br/backup-and-restore-tool.md). For real-time backups of incremental data, refer to [TiCDC](/ticdc/ticdc-overview.md).
+
+When using Dumpling, you need to execute the export command on a running cluster. This document assumes that there is a TiDB instance on the `127.0.0.1:4000` host and that this TiDB instance has a root user without a password.
+
+## Export data from TiDB
+
+Export data using the following command:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+dumpling \
+  -u root \
+  -P 4000 \
+  -H 127.0.0.1 \
+  --filetype sql \
+  --threads 32 \
+  -o /tmp/test \
+  -F $(( 1024 * 1024 * 256 ))
+```
+
+In the above command, `-H`, `-P` and `-u` mean address, port and user, respectively. If password authentication is required, you can pass it to Dumpling with `-p $YOUR_SECRET_PASSWORD`.
+
+Dumpling exports all tables (except for system tables) in the entire database by default. You can use `--where <SQL where expression>` to select the records to be exported. If the exported data is in CSV format (CSV files can be exported using `--filetype csv`), you can also use `--sql <SQL>` to export records selected by the specified SQL statement. 
+
+For example, you can export all records that match `id < 100` in `test.sbtest1` using the following command:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+./dumpling \
+  -u root \
+  -P 4000 \
+  -H 127.0.0.1 \
+  -o /tmp/test \
+  --filetype csv \
+  --sql "select * from `test`.`sbtest1` where id < 100"
+```
+
+Note that the `--sql` option can be used only for exporting CSV files for now. However, you can use `--where` to filter the rows to be exported, and use the following command to export all rows with `id < 100`:
+
+> **Note:**
+>
+> You need to execute the `select * from <table-name> where id < 100` statement on all tables to be exported. If any table does not have the specified field, then the export fails.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+./dumpling \
+  -u root \
+  -P 4000 \
+  -H 127.0.0.1 \
+  -o /tmp/test \
+  --where "id < 100"
+```
+
+> **Note:**
+> 
+> Currently, Dumpling does not support exporting only certain tables specified by users (i.e. `-T` flag, see [this issue](https://github.com/pingcap/dumpling/issues/76)). If you do need this feature, you can use [MyDumper](/backup-and-restore-using-mydumper-lightning.md) instead.
+
+The exported file is stored in the `. /export-<current local time>` directory by default. Commonly used parameters are as follows:
```

Comment:
Seems here shouldn't be a space, does it presents at Chinese version of document too? If does, sorry about it.

```suggestion
The exported file is stored in the `./export-<current local time>` directory by default. Commonly used parameters are as follows:
```

Reply:
No, it's my fault. Updated.